plugins/fabric8-multi-tenant-manager/src/main/java/com/redhat/che/multitenant/Fabric8OpenShiftProjectFactory.java

```diff
@@ -69,6 +72,26 @@ public String evaluateLegacyNamespaceName(NamespaceResolutionContext resolutionC
     return evaluateNamespace();
   }
 
+  @Override
+  protected String evalPlaceholders(String namespace, NamespaceResolutionContext ctx) {
+    return evalPlaceholders();
+  }
+
+  @Override
+  protected String evalPlaceholders(String namespace, Subject currentUser, String workspaceId) {
+    return evalPlaceholders();
+  }
+
+  private String evalPlaceholders() {
+    String placeholder = null;
+    try {
+      placeholder = evaluateNamespace();
+    } catch (InfrastructureException e) {
+      LOG.error("Failed to evaluate placeholder for '*-che' namespace: {}", e.getMessage());
+    }
+    return placeholder;
+  }
+
   private String evaluateNamespace() throws InfrastructureException {
     Subject currentSubject = EnvironmentContext.getCurrent().getSubject();
```

Comment:
It makes sense to check is the current user is not ANONYMOUS, if so - please log error along with stacktrace.
It could help investigate the issue with `Service Account does not have permissions to clean up workspaces resources`

Reply:
I think it is done on the deeper level

```throw new InfrastructureException(
          "The anonymous subject is used, and won't be able to perform this action");```